.circleci/src/commands/restore_ruby_cache.yml

```diff
@@ -5,12 +5,14 @@ parameters:
     default: android
 steps:
   - restore_cache:
-      key: 1-gems-{{ checksum "<< parameters.directory >>/Gemfile.lock" }}
+      keys:
+        - 3-gems-{{ arch }}-{{ checksum "<< parameters.directory >>/Gemfile.lock" }}
+        - 3-gems-{{ arch }}-
   - run:
       name: Install Fastlane
-      command: bundle check || bundle install --path vendor/bundle
```

Comment:
Why did you remove the path argument?

Reply:
Not necessary. In Integreat it is also without.